.eleventy.js

```diff
@@ -36,9 +54,29 @@ module.exports = function (config) {
     svgShortcode: 'usa_icons'
   });
 
+  config.addPlugin(syntaxHighlight, {
+
+  });
+
+  config.addPlugin(pluginTOC, {
+    tags: ['h2']
+  });
+
+  config.addPlugin(pluginMermaid, {
+    mermaid_js_src: '/assets/mermaid.min.js',
```

Comment:
is mermaid loading on every page? i don't know how often it is used across the site, but doesn't seem like all that often